### ISSUE TITLE
ERROR: Dust Levels!

### ISSUE THREAD

**Issue by Ninio1973** [taniman/profit-trailer#640]
Hello there!

I'm receiving this error message continually in log.
I tried a fresh install, deleted ProfitTrailerData and the error persists.

2018-01-10 21:13:52 ERROR ProfitTrailerService - Tradehistory - Please fix your dust levels in configuration.properties
2018-01-10 21:13:52 ERROR ProfitTrailerService - Error occured in TH, message null, result: null

my configuration file:

MARKET = BTC

ALL_trading_enabled = true
ALL_enabled_pairs = BTC-SC, BTC-ADA, BTC-XLM, BTC-DNT, BTC-XRP, BTC-STEEM

#Disable Pairs
BTC-PAY_trading_enabled = false
BTC-RDD_trading_enabled = false
BTC-XVG_trading_enabled = false
BTC-RBY_trading_enabled = false
BTC-LTC_trading_enabled = false
BTC-BTG_trading_enabled = false
BTC-ARDR_trading_enabled = false
BTC-ZCL_trading_enabled = false

#boughtprice
BTC-XVG_override_bought_price = 0.00000130
BTC-LTC_override_bought_price = 0.01705900
BTC-OK_override_bought_price = 0.00004800
BTC-BTG_override_bought_price = 0.00005900
BTC-ARDR_override_boughtprice= 0.00010499
BTC-RBY_override_boughtprice= 0.00013499

ALL_max_cost = 0.050
ALL_min_buy_balance = 0
ALL_min_buy_volume = 500
ALL_min_buy_price = 0
ALL_max_trading_pairs = 10
ALL_max_buy_spread = 1

ALL_buy_strategy = EMASPREAD
ALL_buy_value = -0.50
ALL_trailing_buy = 0.3

ALL_sell_strategy = GAIN
ALL_sell_value = 1
ALL_min_profit = 1
ALL_trailing_profit = 0.5

All_stop_loss_trigger = 0
ALL_stop_loss_timeout = 0
ALL_panic_sell_enabled = false

ALL_DCA_enabled = false

ALL_sell_only_mode = false

ALL_btc_price_drop_trigger = 0
ALL_btc_price_drop_recover_trigger = 0

ALL_btc_price_rise_trigger = 0
ALL_btc_price_rise_recover_trigger = 0

ALL_consecutive_buy_trigger = 10
ALL_consecutive_sell_trigger = 5

ALL_pending_order_wait_time = 0
ALL_combined_cancel_pending_trigger = 0

**Comment by Ninio1973** [comment]
#sell options
sell_fillOrKill = false
sell_immediateOrCancel = true

BTC_dust = 0.0011

hideDust = true

testMode=false

#buy options
buy_fillOrKill = false
buy_immediateOrCancel = true

#BTC_dust
BTC_dust = 0.0011

hideDust = true

testMode=false

**Comment by Ninio1973** [comment]
Thank you taniman!

**Comment by Koldobika** [comment]
Hi Ninio1973, I have the same Error, how did you solve it? , I use Binance and I elevated de BTC_dust= 0.0021   but doesnt work.  
thanks

**Comment by Ninio1973** [comment]
Hello Koldobika,

I don't use Binance. I've installed the latest version and the problem has been solves since!  I'd say try reading Gnome-Crypto settings guide there is also one for binance. Maybe it would help. P.S. I've noticed receiving this error message does not affect the bot's trading... so if it is trading normally for you I wouldn't bother ( I know it's irritaing but...) Well, sorry I couldn't be much of a help. Good luck.
